src/services/api.js

```diff
@@ -71,14 +71,18 @@ export default function SDKWrapper() {
   this.pillarWalletSdk = null;
 }
 
-SDKWrapper.prototype.init = function (privateKey: string, updateOAuth?: ?Function, oAuthTokensStored?: ?OAuthTokens) {
+SDKWrapper.prototype.init = function (
+  updateOAuth?: ?Function,
+  oAuthTokensStored?: ?OAuthTokens,
+  tokensFailed?: ?Function,
+) {
   this.pillarWalletSdk = new PillarSdk({
-    privateKey: privateKey.slice(2),
     apiUrl: SDK_PROVIDER, // ONLY if you have platform running locally
     notificationsUrl: NOTIFICATIONS_URL,
     investmentsUrl: INVESTMENTS_URL,
     updateOAuthFn: updateOAuth,
     oAuthTokens: oAuthTokensStored,
+    tokensFailedCallbackFn: tokensFailed,
```

Comment:
Maybe it's better to call `onTokensExpired`? Or they (tokens) could fail because of other reasons?

Reply:
They can actually if they are revoked or signature cert has changed, I can rename it though to onTokensFailed, ok?